roles/open-jdk/jdk/templates/java_script.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "setting java environment path"
+export JAVA_HOME=/usr/lib/jvm/jdk1.8.0_60

```

Comment:
This should be based on the version variable. change this to use the jdk version